mobile-app/app/screens/AppNavigator/screens/Portfolio/components/PortfolioCard.tsx

```diff
@@ -11,16 +11,15 @@ import { tailwind } from '@tailwind'
 import { ButtonGroup } from '../../Dex/components/ButtonGroup'
```

Comment:
require cycle warning between this component and `EmptyTokensScreen.tsx`

```import { ButtonGroupTabKey } from './PortfolioCard'``` might be this